contracts/BondedFungibleToken.sol

```diff
@@ -145,7 +150,7 @@ contract BondedFungibleToken is Initializable, BFTEvents, Ownable, BancorFormula
         );
     }
 
-    function sell(uint256 _toSell, uint256 _expected, uint256 _maxSlippage)
+    function sell(uint256 _toSell) // this doesn't need expected & slippage
```

Comment:
It does if we want to revert if the user will get back less than they want.

Reply:
true, same as for buy.. changed the comment to remind us to add this back in